pkg/controller/knativeserving/common/helpers.go

```diff
@@ -34,3 +40,55 @@ func UpdateConfigMap(cm *unstructured.Unstructured, data map[string]string, log
 		unstructured.SetNestedField(cm.Object, v, "data", k)
 	}
 }
+
+// Set some data in a configmap, only overwriting common keys if they differ
```

Comment:
Is this comment leftover from somewhere?

I'd also expect it to start with // UpdateImage

Reply:
Done